src/codewhisperer/activation.ts

```diff
@@ -64,6 +65,28 @@ export async function activate(context: ExtContext): Promise<void> {
         await enableDefaultConfigCloud9()
     }
 
+    //This logic is to show the Getting Started Page only once for the first time SignIn Users
+    let start = ''
+
+    const hasShownWelcomeMsgBefore = get(
```

Comment:
Can you move line 68 to line 88 to a function named `tryShowGettingStartedPage`?

Then in this function, clearly call out that `hasShownWelcomeMsgBefore` is for users who have signed in before v1.91.0. Make it clear that `hasShownWelcomeMsgBeforeToExistingUsers` is for users who has seen or not see the new getting started experience since v1.92.0

Reply:
Updated accordingly.